tests/cypress/integration/frontend/split-content.spec.js

```diff
@@ -0,0 +1,82 @@
+describe('Block visibility tests for non WM-enabled users', () => {
+	beforeEach(() => {
+		cy.visit('/block-visibility/')
+	})
+
+	it('Checks block visibility settings for blocks hidden to non WM-enabled users', () => {
+		cy
+			.get('.wm-shown h3')
+			.invoke('text')
+			.should('contain', 'This content is for Coil Members only. To access, join Coil and install the browser extension.')
```

Comment:
Same here. Move the repeated use of string into a const.

Reply:
It has been, the other PR's need to be merged first